paging/paging-common/src/main/kotlin/androidx/paging/PagingDataDiffer.kt

```diff
@@ -300,7 +317,7 @@ public abstract class PagingDataDiffer<T : Any>(
      * @sample androidx.paging.samples.refreshSample
      */
     public fun refresh() {
-        receiver?.refresh()
+        statelessReceiver?.refresh()
```

Comment:
For when we drop these signals on the floor, I wonder if we should throw an exception. Can we file a bug to follow up on that as a new FR?

Reply:
zzz had to file it under my personal email, b/234749375